casual/casual-api/src/main/java/se/laz/casual/api/buffer/type/fielded/EncodingInfo.java

```diff
@@ -7,23 +7,24 @@
 package se.laz.casual.api.buffer.type.fielded;
 
 import java.nio.charset.Charset;
-import java.util.logging.Logger;
+
+import static java.lang.System.Logger.Level.*;
```

Comment:
As with casual-caller - we have a preference for not using wildcard imports. Please update accordingly in all files.